content/hi/docs/reference/glossary/annotation.md

```diff
@@ -0,0 +1,17 @@
+---
+title: एनोटेशन (Annotation)
+id: annotation
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/annotations
+short_description: >
+  एक की-वैल्यू पेयर जिसका उपयोग मनमाने ढंग से गैर-पहचान वाले मेटाडेटा को ऑब्जेक्ट से जोड़ने के लिए किया जाता है।
+
+aka: 
+tags:
+- fundamental
+---
+ एक की-वैल्यू पेयर जिसका उपयोग मनमाने ढंग से गैर-पहचान वाले मेटाडेटा को ऑब्जेक्ट से जोड़ने के लिए किया जाता है।
+
+<!--more--> 
+
+एनोटेट किया गया मेटाडेटा छोटा या बड़ा, संरचित या असंरचित हो सकता है, और इसमें ऐसे वर्ण हो सकते हैं जिनकी {{< glossary_tooltip text="लेबल" term_id="label" >}} में अनुमति नहीं है। आप क्लाइंट जैसे टूल और लाइब्रेरी के साथ मेटाडेटा पुनर्प्राप्त कर सकते हैं।
```

Comment:
```suggestion
एनोटेट किया गया मेटाडेटा छोटा या बड़ा, संरचित या असंरचित हो सकता है, और इसमें ऐसे वर्ण हो सकते हैं जिनकी {{< glossary_tooltip text="लेबल" term_id="label" >}} में अनुमति नहीं है। आप क्लाइंट जैसे टूल और लाइब्रेरी के द्वारा मेटाडेटा पुनर्प्राप्त कर सकते हैं।
```

Reply:
@anubha-v-ardhan The change has been applied.